public/index.html

```diff
@@ -10,6 +10,7 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <script src="./TW-ELEMENTS-PATH/dist/js/index.min.js"></script>
```

Comment:
Do we really need this script to use tailwind? If I recall correctly we do not. Can you remove this and check? If it is not needed please delete it.